package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gulp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
please add start script here (that will run default gulp task):
"start": "gulp"